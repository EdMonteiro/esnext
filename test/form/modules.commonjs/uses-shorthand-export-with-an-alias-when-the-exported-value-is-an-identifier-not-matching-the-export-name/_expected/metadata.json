{
  "modules.commonjs": {
    "imports": [],
    "exports": [
      {
        "type": "named-export",
        "bindings": [
          {
            "exportName": "b",
            "localName": "a"
          }
        ],
        "node": {
          "type": "ExpressionStatement",
          "expression": {
            "type": "AssignmentExpression",
            "operator": "=",
            "left": {
              "type": "MemberExpression",
              "object": {
                "type": "Identifier",
                "name": "exports",
                "typeAnnotation": null
              },
              "property": {
                "type": "Identifier",
                "name": "b",
                "typeAnnotation": null
              },
              "computed": false
            },
            "right": {
              "type": "Identifier",
              "name": "a",
              "typeAnnotation": null
            }
          }
        }
      }
    ],
    "directives": []
  }
}