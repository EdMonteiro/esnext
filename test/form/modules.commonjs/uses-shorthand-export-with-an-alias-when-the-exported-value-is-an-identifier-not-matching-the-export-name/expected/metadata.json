{
  "modules.commonjs": {
    "imports": [],
    "exports": [
      {
        "type": "named-export",
        "bindings": [
          {
            "exportName": "b",
            "localName": "a"
          }
        ],
        "node": {
          "type": "ExpressionStatement",
          "expression": {
            "type": "AssignmentExpression",
            "operator": "=",
            "left": {
              "type": "MemberExpression",
              "computed": false,
              "object": {
                "type": "Identifier",
                "name": "exports"
              },
              "property": {
                "type": "Identifier",
                "name": "b"
              }
            },
            "right": {
              "type": "Identifier",
              "name": "a"
            }
          }
        }
      }
    ],
    "directives": []
  }
}